src/Town/components-edit/EditSociopolitics.twee

```diff
@@ -1,5 +1,5 @@
 :: EditSociopolitics
-<span class="tip" title="How many people live here?">Town Population: <<numberslider "$town.population" $town.population 30 12000 10>></span>
+<<liveblock>><<run setup.updateDemographics($town, $town._baseDemographics)>><span class="tip" title="How many people live here?" ontouchend="$('.race-slider').trigger(':liveupdate')" onmouseup="$('.race-slider').trigger(':liveupdate')">Town Population: <<numberslider "$town.population" $town.population 30 12000 10>></span><</liveblock>>
```

Comment:
I feel like there might be a memory leak in here. Could we give this element an ID, and add the event handlers via function call and `addEventListener` instead? Should be much clearer then if what's happening is safe or not.

Reply:
`<<liveblock>>` is a shortcut that re-renders, essentially the same as `<<replace "#css-identifier">><<include "PassageThatIWantToInclude">><</replace>>`.

Reply:
Yes, but inline event handlers is never a good idea, for various reasons.

See https://stackoverflow.com/questions/11737873/why-are-inline-event-handler-attributes-a-bad-idea-in-modern-semantic-html